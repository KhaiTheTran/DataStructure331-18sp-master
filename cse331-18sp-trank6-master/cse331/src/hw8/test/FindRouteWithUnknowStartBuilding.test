# Test Unknown Start Building
# Print Unknown building: [building name]

# Start program
r
QFC
EEB

# Terminate Program
q