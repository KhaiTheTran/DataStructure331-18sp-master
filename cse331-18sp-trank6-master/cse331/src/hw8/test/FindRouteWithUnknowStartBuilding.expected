Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Test Unknown Start Building
# Print Unknown building: [building name]

# Start program
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: QFC
Enter an option ('m' to see the menu):
# Terminate Program